sql-jdbc/src/main/java/org/opensearch/jdbc/types/OpenSearchType.java

```diff
@@ -62,8 +62,8 @@ public enum OpenSearchType {
     IP(JDBCType.VARCHAR, String.class, 15, 0, false),
     NESTED(JDBCType.STRUCT, null, 0, 0, false),
     OBJECT(JDBCType.STRUCT, null, 0, 0, false),
-    DATE(JDBCType.DATE, Date.class, 24, 24, false),
-    TIME(JDBCType.TIME, Time.class, 24, 24, false),
+    DATE(JDBCType.DATE, Date.class, 10, 10, false),
+    TIME(JDBCType.TIME, Time.class, 8, 8, false),
     DATETIME(JDBCType.TIMESTAMP, Timestamp.class, 24, 24, false),
     TIMESTAMP(JDBCType.TIMESTAMP, Timestamp.class, 24, 24, false),
```

Comment:
Should we update DATETIME and TIMESTAMP to a precision of `9`?

Reply:
Done, thanks!